### PR TITLE
Adds viro-terror kit and grenades

### PR DESCRIPTION
**Pull request by LanCartwright**
not space aids but god damn you should be scared

### CODE REVIEW

File: code/datums/diseases/tuberculosis.dm [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/datum/disease/Tuberculosis
```
Comment (RemieRichards) [inline code/datums/diseases/tuberculosis.dm]: capital at the start of the path? that's... uncommon.
best to change it for consistency, unless of course the other disease do that in which case wat.

Comment (LanCartwright) [inline code/datums/diseases/tuberculosis.dm]: I'll drop it to lower case. I don't know how that got there.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -112,3 +112,11 @@
 	name = "morphine medipen"
 	desc = "A rapid way to get you out of a tight situation and fast! You'll feel rather drowsy, though."
 	list_reagents = list("morphine" = 10)
+
+/obj/item/weapon/reagent_containers/hypospray/medipen/Tuberculosiscure
```
Comment (RemieRichards) [inline code/modules/reagents/reagent_containers/hypospray.dm]: AHHH CAPITALS AGAIN

File: code/datums/diseases/tuberculosis.dm [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+				affected_mob << "<span class='userdanger'>[pick("You feel your heart slowing...", "You slow your heartbeat.")]</span>"
+				affected_mob.adjustStaminaLoss(40)
+				if(affected_mob.getStaminaLoss() > 60 && !M.stat)
+					affected_mob.visible_message("<span class='warning'>[M] faints!</span>", "<span class='userdanger'>You surrender yourself and feel at peace...</span>")
+					affected_mob.sleeping += 5
+			if(prob(2))
+				affected_mob << "<span class='userdanger'>You feel your mind relax and your thoughts drift!</span>"
+				affected_mob.confused = min(100, M.confused + 8)
+			if(prob(15)
+				/datum/symptom/vomit/blood/Vomit(mob/living/carbon/M)
```
Comment (Anonus) [inline code/datums/diseases/tuberculosis.dm]: Remove this, it doesn't do anything here.

File: code/datums/diseases/tuberculosis.dm [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+				affected_mob.emote("cough")
+				affected_mob << "<span class='danger'>Your chest hurts.</span>"
+			if(prob(2))
+				affected_mob << "<span class='danger'>Your stomach violently rumbles!</span>"
+			if(prob(2))
+				affected_mob << "<span class='danger'>You feel a cavity in your lung forming.</span>"
+				affected_mob.adjustOxyLoss(5)
+				affected_mob.emote("gasp")
+			if(prob(5))
+				/datum/symptom/vomit/proc/Vomit(mob/living/carbon/M)
```
Comment (Anonus) [inline code/datums/diseases/tuberculosis.dm]: Remove this, it doesn't do anything here.

**Comment by Shadowlight213** [comment]
Hmm, I would have gone with a vaccine for the ops, since by just having the antidote be a reagent mix they are running into the low cure chance. However if their suits being sealed protects them from the virus it seems like an interesting idea.

**Comment by LanCartwright** [comment]
It shouldn't be a major issue - I hope - since the medipens have two uses.

I'm struggling to figure out what's causing the travis fuckup.

**Comment by Anonus** [comment]
Try fixing the `/T` stuff I don't think byond likes non-procs that start with capitals

**Comment by ike709** [comment]
Why so many commits? Can't you just do a Find and Replace for all of the files and commit that instead of doing a commit for every file you change?

**Comment by LanCartwright** [comment]
i don't know how/I'm really lazy.

I only started coding drunk 3 days ago. My apologies.

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 5]
```diff
@@ -296,4 +296,17 @@
...
\ No newline at end of file
+	spawned_disease = /datum/disease/fluspanish
+
+/obj/item/weapon/reagent_containers/glass/bottle/tuberculosis
+	name = "Fungal Tuberculosis culture bottle"
+	desc = "A small bottle. Contains a sample of Fungal Tubercle bacillus."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/tuberculosis
+
+/obj/item/weapon/reagent_containers/glass/bottle/Tuberculosiscure
```
Comment (Anonus) [inline code/modules/reagents/reagent_containers/bottle.dm]: one more

**Comment by PKPenguin321** [comment]
traitors will use this practically every round, i guarantee it

**Comment by LanCartwright** [comment]
grenade is nuke only and virus kit is counterable

**Comment by LanCartwright** [comment]
reopening this PR with a more upto date code.

**Pull request closed without merging**